transactions.md

```diff
@@ -6,9 +6,61 @@ One transaction event occurs at authorisation time. The other transaction event
 
 ## Transaction Object
 
-##### API version from `2022-07-13`
+| **Field name** | **Description** | **Type** |
+| --- | ------ | --- |
+| `accountId` | The unique identifier of the user account at Fidel API. | string |
+| `amount` | The amount of the transaction in the currency it was charged in. | number |
+| `approvalCode` | Unique code that is sent along with the authorization of the transaction by Amex. It mirrors the identifiers.amexApprovalCode property. | string (or null) |
+| `auth` | Indicates whether Fidel API received an authorization event for this transaction. | boolean |
+| `authCode` | Unique code that is sent along with the authorization of the transaction by Visa and Mastercard. It is sent by the issuer/the PCN on the issuer's behalf when approving a transaction. It mirrors the identifiers.mastercardAuthCode or identifiers.visaAuthCode properties, depending on the issuing card for the transaction. | string (or null) |
+| `brand.id` | The identifier of the brand at Fidel API. | string |
+| `brand.logoURL` | The URL of the logo for the brand. | string (or null) |
+| `brand.name` | The name of the merchant where the purchase was made. | string |
+| `brand.metadata` | Customized information you added to the brand. | object (or null) |
+| `card.firstNumbers` | The first numbers of the card, used for helping users identify their cards. | string |
+| `card.id` | The tokenized card identifier created by Fidel API. | string |
+| `card.lastNumbers` | The last numbers of the card, used for helping users identify their cards. | string |
+| `card.metadata` | Customized information you added to the card. | object (or null) |
+| `card.scheme` | The payment network (Visa, Mastercard or Amex). | string |
+| `cardPresent` | Indicates whether the transaction was in store (as opposed to online). Only available in Visa transactions, and only if the transaction data received from Visa contains this information. In test transactions, this field is always null. | boolean (or null) |
+| `cleared` | Indicates whether Fidel API received a clearing event for this transaction. | boolean |
+| `created` | The time when the transaction object was created in the Fidel API database. | string, in ISO format |
```

Comment:
Do we have the ISO format reference?

Reply:
Like we add on country code

Reply:
Added reference in types with standard formats.